pipenv/cli.py

```diff
@@ -407,6 +417,8 @@ def do_create_virtualenv(three=None, python=None):
             python = 'python2'
         elif three is True:
             python = 'python3'
+        else:
+            python = get_python_from_requires()
```

Comment:
+1 @nateprewitt. I think these are good changes though, and they are welcome. We'll just not want to jump the gun on them. Take it slow :)